File: ring-android/app/src/main/java/cx/ring/tv/main/ChannelActionHandler.java
package cx.ring.tv.main;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class ChannelActionHandler extends BroadcastReceiver {
    private static final String TAG = ChannelActionHandler.class.getSimpleName();

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.w(TAG, "onReceive " + intent);
    }
}

File: ring-android/app/src/main/java/cx/ring/tv/main/ChannelActionHandler.kt
package cx.ring.tv.main

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class ChannelActionHandler : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Log.w(TAG, "onReceive $intent")
    }

    companion object {
        private val TAG = ChannelActionHandler::class.simpleName!!
    }
}